sanic/app.py

```diff
@@ -735,6 +736,18 @@ async def handle_exception(
             context={"request": request, "exception": exception},
         )
 
+        if (
+            request.stream is not None
+            and request.stream.stage is not Stage.HANDLER
+        ):
+            error_logger.exception(exception)
```

Comment:
We should explicitly add `exc_info=True`.

Reply:
Why we would need to explicitly add this? I think `error_logger.exception` has that as the default behavior?

Reply:
Added.

Reply:
I think you're right, but I want to check. I was (and still am) on my phone. Please confirm.